src/WingetCreateCLI/Program.cs

```diff
@@ -25,6 +25,14 @@ private static async Task<int> Main(string[] args)
             Logger.Initialize();
             UserSettings.FirstRunTelemetryConsent();
             TelemetryEventListener.EventListener.IsTelemetryEnabled();
+
+            string latestVersion = await GitHub.GetLatestRelease();
+            if (!latestVersion.Contains(Utils.GetEntryAssemblyVersion()))

```

Comment:
This might be a little too fuzzy, for example:

v11.0.0.0-preview would contain 1.0.0.0

How about:

(latestVersion.TrimStart('v').Split('-').First() == Utils.GetEntryAssemblyVersion() instead?

Reply:
Changed to suggestion, thanks for catching that mistake, can't wait for v11 of wingetcreate!